test/lib/ufe/testCamera.py

```diff
@@ -231,16 +231,73 @@ def _TestProjection(self, ufeCamera, usdCamera):
 
     @unittest.skipUnless(mayaUtils.mayaMajorVersion() >= 2023, 'Requires Python API only available in Maya 2023 or greater.')
     def testUsdCamera(self):
-        self._StartTest('TranslateRotate_vs_xform')
         mayaPathSegment = mayaUtils.createUfePathSegment('|stage|stageShape')
         cam2UsdPathSegment = usdUtils.createUfePathSegment('/cameras/cam2/camera2')
-        cam2Path = ufe.Path([mayaPathSegment, cam2UsdPathSegment])
-        cam2Item = ufe.Hierarchy.createItem(cam2Path)
+        camera2Path = ufe.Path([mayaPathSegment, cam2UsdPathSegment])
+        cam2Item = ufe.Hierarchy.createItem(camera2Path)
 
         cam2Camera = ufe.Camera.camera(cam2Item)
         cameraPrim = usdUtils.getPrimFromSceneItem(cam2Item)
         self._TestCameraAttributes(cam2Camera, cameraPrim)
 
+    @unittest.skipUnless(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') >= '4011', 'Requires Ufe Preview Version at least 2011.')
+    def testUsdCameraHandler(self):
+        mayaPathSegment = mayaUtils.createUfePathSegment('|stage|stageShape')
+        cam2UsdPathSegment = usdUtils.createUfePathSegment('/cameras/cam2/camera2')
+        camera2Path = ufe.Path([mayaPathSegment, cam2UsdPathSegment])
+
+        # Test that the camera handlers can find USD cameras in a scene segment
+        proxyShapeParentSegment = mayaUtils.createUfePathSegment('|stage')
+        camerasParentPathSegment = usdUtils.createUfePathSegment('/cameras')
+        camera2ParentPathSegment = usdUtils.createUfePathSegment('/cameras/cam2')
+        geoPathSegment = usdUtils.createUfePathSegment('/GEO')
+        camera1PathSegment = usdUtils.createUfePathSegment('/cameras/cam1/camera1')
+        
+        proxyShapePath = ufe.Path([mayaPathSegment])
+        proxyShapeParentPath = ufe.Path([proxyShapeParentSegment])
+        camerasParentPath = ufe.Path([mayaPathSegment, camerasParentPathSegment])
+        camera2ParentPath = ufe.Path([mayaPathSegment, camera2ParentPathSegment])
+        geoPath = ufe.Path([mayaPathSegment, geoPathSegment])
+        camera1Path = ufe.Path([mayaPathSegment, camera1PathSegment])
+
+        proxyShapeItem = ufe.Hierarchy.createItem(proxyShapePath)
```

Comment:
Have a look at ufeUtils.createItem().  It will create a ufe.SceneItem directly out of a path string, I think this is what you want, and will remove a lot of boiler-plate code from your test.